<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-G9L5PQ2" failures="1" tests="2" name="Default test(failed)" time="38.75" errors="0" timestamp="2023-01-13T22:26:10 IST">
  <testcase classname="TestCases.LoginPageTest" name="veriFyBotLogoTest" time="0.081"/>
  <testcase classname="TestCases.LoginPageTest" name="verifyLoginLogoTest" time="0.083">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestCases.LoginPageTest.verifyLoginLogoTest(LoginPageTest.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginLogoTest -->
  <testcase classname="TestCases.LoginPageTest" name="loginToAppTest" time="0.0">
    <ignored/>
  </testcase> <!-- loginToAppTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyTitleTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyTitleTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyCurrentUrlTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyCurrentUrlTest -->
</testsuite> <!-- Default test(failed) -->
